pom.xml

```diff
@@ -2297,7 +2297,7 @@
             <dependency>
                 <groupId>org.postgresql</groupId>
                 <artifactId>postgresql</artifactId>
-                <version>42.7.4</version>
+                <version>42.7.5</version>
```

Comment:
@Praveen2112 Could you look into TestPostgreSqlConnectorTest failures and send a new PR?